lib/job_queue/job_node.cpp

```diff
@@ -281,7 +283,9 @@ job_queue_node_type * job_queue_node_alloc( const char * job_name ,
   UTIL_TYPE_ID_INIT( node , JOB_QUEUE_NODE_TYPE_ID );
 
   /* The data initialized in this block should *NEVER* change. */
-  node->job_name       = util_alloc_string_copy( job_name );
+  std::string path     = job_name;
+  std::string basename = path.substr(path.find_last_of("/\\") + 1);
```

Comment:
What happens if there is no `/` in the string?

Reply:
Then std::string_path = job_name